Resources/Markdown/docs/builds.md

```diff
@@ -5,21 +5,25 @@ description: Frequently Asked Questions about the Swift Package Index Build Syst
 
 ## The Swift Package Index Build System
 
-* [What is the Build System?](#build-system)
-* [Why does a package show missing or incomplete compatibility?](#no-builds)
-* [What revision is the default branch built for?](#what-revision)
-* [How are packages built?](#built-how)
-* [How does the build system build with different Swift versions?](#swift-versions)
-* [When was a package last built?](#last-built)
-* [Is it possible to hide failing builds for unsupported platforms?](#hide-failing-builds)
-* [If a build is showing failed incorrectly, how can I fix it?](#fix-false-negative)
-* [If a build is showing an error that seems unrelated to the build, how can I fix it?](#unrelated-error)
+- [What is the Swift Package Index Build System?](#build-system)
+- [Why does a package show missing or incomplete compatibility?](#no-builds)
+- [What revision is the default branch built for?](#what-revision)
+- [How are packages built?](#built-how)
+- [How does the build system build with different Swift versions?](#swift-versions)
+- [When was a package last built?](#last-built)
+- [Is it possible to hide failing builds for unsupported platforms?](#hide-failing-builds)
+- [If a build is showing failed incorrectly, how can I fix it?](#fix-false-negative)
+- [If a build is showing an error that seems unrelated to the build, how can I fix it?](#unrelated-error)
 
-<h3 id="build-system">What is the Build System?</h3>
+<h3 id="build-system">What is the Swift Package Index Build System?</h3>
 
-A Swift package manifest includes support for declaring [Swift version support](https://developer.apple.com/documentation/swift_packages/package/3197887-swiftlanguageversions) and [platform support](https://developer.apple.com/documentation/swift_packages/package/3197886-platforms). There are a few problems with relying on this data, so the Swift Package Index takes a different approach. What better way to determine compatibility with a specific version of Swift on a particular platform than to build it?
+Rather than rely on the [Swift language versions](https://developer.apple.com/documentation/swift_packages/package/3197887-swiftlanguageversions) and [platform support](https://developer.apple.com/documentation/swift_packages/package/3197886-platforms) fields in the package manifest, the Swift Package Index bases its Swift version and platform compatibility data on real-world results from our build system.
```

Comment:
I don't think we should be mentioning the platform manifest here - it's misleading and wouldn't really work as an indicator.

Reply:
Just realised this was in there before, never mind. (It's a terribly confusing field...)

Reply:
Yea. That was the whole reason I re-worded this section. We led with information about this before.

You're right though, we shouldn't even mention it. I'll amend.